src/OpenTelemetry.Api/Metrics/LabelSet.cs

```diff
@@ -29,11 +32,53 @@ public class LabelSet
         public LabelSet(IEnumerable<KeyValuePair<string, string>> labels)
         {
             this.Labels = labels;
+            this.LabelSetEncoded = this.GetLabelSetEncoded(labels);
         }
 
         /// <summary>
         /// Gets or sets the labels for this LabelSet.
         /// </summary>
         public IEnumerable<KeyValuePair<string, string>> Labels { get; set; }
+
+        /// <summary>
+        /// Gets or sets the label set in encoded form for this LabelSet.
+        /// </summary>
+        public string LabelSetEncoded { get; set; }
```

Comment:
should it be public?

Reply:
This whole class implementation will be moved from API to SDK in a subsequent pr.